inc/rest.php

```diff
@@ -423,6 +435,63 @@ public function get_sample_rate( WP_REST_Request $request ) {
 		return $this->response( $image );
 	}
 
+	/**
+	 * Crawl & upload initial load.
+	 *
+	 * @return boolean True if no more posts left to receive, false if there are more left.
```

Comment:
This response should be `WP_REST_Response` , it's not breaking anything but just to be accurate.